apps/timetable/admin/ui/css/admin.css

```diff
@@ -295,3 +300,72 @@
 #gh-borrow-series-modal .list-group .list-group-item .gh-borrow-series-borrowed i.fa-check {
     display: block;
 }
+
+
+/*****************/
+/* BATCH EDITING */
+/*****************/
+
+#gh-batch-edit-container {
+    margin: 30px;
+}
+
+#gh-batch-edit-container h1 {
+    font-size: 25px;
+    font-weight: 400;
+}
+
+#gh-batch-edit-header > div > * {
+    height: 39px;
+    width: 100%;
+}
+
+.gh-batch-edit-events-container {
+    margin-bottom: 100px;
+}
+
+.gh-batch-edit-events-container .table > thead > tr > th {
+    border-bottom-width: 5px;
+}
+
+.gh-batch-edit-events-container .table > tbody > tr > td {
+    max-width: 200px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    vertical-align: middle;
+    white-space: nowrap;
+}
+
+.gh-batch-edit-events-container .table > tbody > tr.info td {
+    background-color: #ECF5F9;
+}
+
+.gh-batch-edit-events-container .table > tbody > tr > td:first-child {
+    max-width: 40px;
+}
+
+.gh-batch-edit-events-container .table > thead > tr > th .checkbox,
+.gh-batch-edit-events-container .table > tbody > tr > td .checkbox {
+    margin: 0;
+}
+
+.gh-batch-edit-events-container .table > thead > tr > th .checkbox input,
+.gh-batch-edit-events-container .table > tbody > tr > td .checkbox input {
+    margin-top: 2px;
+}
+
+.gh-batch-edit-actions-container {
+    background: #FFF;
+    bottom: 0;
+    border-top: 1px solid #EEE;
+    -webkit-box-shadow: 0 6px 37px;
+       -moz-box-shadow: 0 6px 37px;
+            box-shadow: 0 6px 37px;
+    margin-left: -30px;
+    padding: 20px;
+    position: fixed;
+    width: -moz-calc(100% - 320px); /* Firefox */
+    width: -webkit-calc(100% - 320px); /* WebKit */
+    width: -o-calc(100% - 320px); /* Opera */
+    width: calc(100% - 320px); /* Standard */
```

Comment:
I -think- that I will need to leave the standardised `calc` at the bottom where it is so that the browsers that do support it override it properly.